### PR TITLE
correction

### PR DESCRIPTION
**Pull request by meghasharma123**
I removed that i , I think that save some spaces 👍

### CODE REVIEW

File: Turbo Sort TSORT.c [hunk 1]
```diff
@@ -5,7 +5,7 @@ int arr[1000000]={0};
      
 void main()
 {
-	int a, i,n;
+	int a,n;//See here in this code i is useless so i think that can be deleted.
```
Comment (coderbond007) [inline Turbo Sort TSORT.c]: No need to add this comment here. add this comment to the pull request instead.

**Review by coderbond007 — COMMENTED** [review]
remove comment

File: Turbo Sort TSORT.c (hunk 1 above)
Comment (meghasharma123) [reply Turbo Sort TSORT.c]: Yes yes...

**Review by meghasharma123 — COMMENTED** [review]
Done

**Comment by coderbond007** [comment]
LGTM, merging to master.

**Comment by coderbond007** [comment]
@meghasharma123 add the solution to new questions rather than modifying new ones.

**Comment by meghasharma123** [comment]
👍🙂

**Pull request merged**